mirror/dependencies/pnda-rpm-package-dependencies.txt

```diff
@@ -30,13 +30,15 @@ MySQL-python-1.2.5-1.el7
 nginx
 nmap-ncat
 ntp-4.2.6p5
+patch
```

Comment:
patch need to be part of deb dependencies

Reply:
Added